Dockerfile

```diff
@@ -1,64 +1,25 @@
-FROM ubuntu:18.04 as build
+FROM ubuntu:20.04 as build
+
+ENV DEBIAN_FRONTEND=noninteractive
 
 RUN apt-get update && \
```

Comment:
It would probably be better to run `./install.sh --extra` rather than hardcode libraries here